phpBB/phpbb/avatar/driver/upload.php

```diff
@@ -13,15 +13,17 @@
 
 namespace phpbb\avatar\driver;
 
+use \phpbb\storage\exception\exception as storage_exception;
```

Comment:
keep it like that, I don't mind.
It's just that I'm starting to ask my self if we shouldn't suffix every class (every where in core) because at some point exception or local is not really meaningful... anyway, do not change anything :)